src/main/java/teammates/common/datatransfer/InstructorAttributes.java

```diff
@@ -28,13 +29,8 @@
     public String role;
     public boolean isDisplayedToStudents;
     public String displayedName;
-    
-    /**
-     * The json representation of privileges, used for storing the 
-     * instructorPrivilege content in instructor entity
-     */
-    public String instructorPrivilegesAsText;
-    public transient InstructorPrivileges privileges;
+
+    public InstructorPrivileges privileges;
```

Comment:
This variable was `transient` before. Any side effect for no longer making it `transient`?